### PR TITLE
Deployment script now have setup of voting apps and of Api3Dao instance

### LINKED ISSUE #179: Add: `setDaoApps()` to the deployment script
Pool contract's `setDaoApps()` should be called as a part of the deployment script

### PR DESCRIPTION
**Pull request by BritikovKI**
Closes #179

### CODE REVIEW

File: packages/dao/scripts/deploy.js [hunk 1]
```diff
@@ -2,16 +2,29 @@
...
 
+const Api3Pool = artifacts.require('Api3Pool');
+
+const SUPPORT_1 = 80e16;
+const ACCEPTANCE_1 = 40e16;
+const VOTING_DURATION_1 = 100;
+
+const SUPPORT_2 = 50e16;
+const ACCEPTANCE_2 = 20e16;
+const VOTING_DURATION_2 = 60;
```
Comment (bbenligiray) [inline packages/dao/scripts/deploy.js:19]:
```
const SUPPORT_1 = 50e16;
const ACCEPTANCE_1 = 50e16;
const VOTING_DURATION_1 = 7 * 24 * 60 * 60;

const SUPPORT_2 = 50e16;
const ACCEPTANCE_2 = 15e16;
const VOTING_DURATION_2 = 7 * 24 * 60 * 60;
```

Comment (BritikovKI) [reply packages/dao/scripts/deploy.js]: Done

**Comment by bbenligiray** [comment]
I'm not comfortable with reviewing deployment stuff

File: packages/dao/scripts/deploy.js [hunk 2]
```diff
@@ -27,6 +40,21 @@ module.exports = async callback => {
                 {name: 'payroll', contractName: 'Payroll'},
                 {name: 'finance', contractName: 'Finance'},
                 {name: 'token-manager', contractName: 'TokenManager'}]);
+        const tx = await template.newInstance('api3template', (api3Pool.address), [SUPPORT_1, ACCEPTANCE_1, VOTING_DURATION_1], [SUPPORT_2, ACCEPTANCE_2, VOTING_DURATION_2])
```
Comment (kinspotting) [inline packages/dao/scripts/deploy.js:43]: pls change 'api3template' to `TEMPLATE_NAME`

**Review by kinspotting — CHANGES_REQUESTED** [review]

File: packages/dao/scripts/deploy.js (hunk 2 above)
Comment (BritikovKI) [reply packages/dao/scripts/deploy.js]: Done

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**